sql/core/src/test/scala/org/apache/spark/sql/TPCDSQueryTestSuite.scala

```diff
@@ -131,16 +132,44 @@ class TPCDSQueryTestSuite extends QueryTest with TPCDSBase with SQLQueryTestHelp
     }
 
     assertResult(expectedSchema, s"Schema did not match\n$queryString") { schema }
-    assertResult(expectedOutput, s"Result did not match\n$queryString") { outputString }
+    if (needSort) {
+      val expectSorted = expectedOutput.split("\n").sorted.map(_.trim)
+        .mkString("\n").replaceAll("\\s+$", "")
+      val outputSorted = output.sorted.map(_.trim).mkString("\n").replaceAll("\\s+$", "")
+      assertResult(expectSorted, s"Result did not match\n$queryString") { outputSorted }
+    } else {
+      assertResult(expectedOutput, s"Result did not match\n$queryString") { outputString }
+    }
   }
 
+  val broadcastHashJoinConf = Map(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "10485760")
```

Comment:
why do we pick `10485760`?

Reply:
I want to keep consistent with the configuration in the `sql/core/src/test/resources/sql-tests/inputs/postgreSQL/join.sql`